backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHShutdownApi.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.glutenproject.backendsapi.clickhouse
+
+import io.glutenproject.backendsapi.ShutdownApi
+import io.glutenproject.vectorized.JniLibLoader
+
+class CHShutdownApi extends ShutdownApi {
+  override def shutdown(): Unit = {
+    val loaded = JniLibLoader.REQUIRE_UNLOAD_LIBRARY_PATHS.toArray.reverse
+    loaded.foreach(
+      path => {
+        JniLibLoader.unloadFromPath(path.asInstanceOf[String])
+      })
+  }
```

Comment:
Can we add a static method `JniLibLoader#forceUnloadAll` and call the method from here?

The built-in shutdown hook in JniLibLoader can call the same method as well.

Reply:
@zhztheplayer Does gluten with velox has the same issue?

Reply:
Not yet. But it seems to be a graceful way for Velox backend as well, to unload the libs on exit.

However it may require some tests since I remember last time we try unloading in Velox/Arrow backend something crashes the process.